src/compiler/watch.ts

```diff
@@ -48,6 +48,14 @@ namespace ts {
         };
     }
 
+    export function createWatchDiagnosticReporterWithColor(system = sys): DiagnosticReporter {

```

Comment:
Hmm, I just saw this. It's unfortunate that we need a new reporter. This should probably be marked `/** @internal */` though.